packages/mui-material/src/ListItemButton/ListItemButton.js

```diff
@@ -177,7 +177,8 @@ const ListItemButton = React.forwardRef(function ListItemButton(inProps, ref) {
       <ListItemButtonRoot
         ref={handleRef}
         href={other.href || other.to}
-        component={(other.href || other.to) && component === 'div' ? 'a' : component}
+        // BaseButton processes href when component is set to 'button'
```

Comment:
```suggestion
        // `ButtonBase` processes `href` or `to` iff `component` is set to 'button'
```

Reply:
I've updated the comment according to your suggestion